plutus-tx-plugin/test/Plugin/Data/polymorphic/polyConstructed.pir.golden

```diff
@@ -11,8 +11,37 @@
         (vardecl Poly (fun a [ [ MyPolyData a ] b ]))
       )
     )
+    (termbind
+      (strict)
+      (vardecl
+        `$WPoly`
```

Comment:
This *should* get inlined in the end, but of course we can't tell because we don't have simplificaiton on here